### PR TITLE
11.0 mig partner address street3

### PR DESCRIPTION
**Pull request by huyly0909**

### CODE REVIEW

File: partner_address_street3/model/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import partner
+from . import country
```
Comment (hieulucky111) [inline partner_address_street3/model/__init__.py]: Import Order? https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Review by hieulucky111 — APPROVED** [review]
Some minor adjustments, LGTM overall.

**Review by jcdrubay — APPROVED** [review]
LGTM

**Comment by jcdrubay** [comment]
@OCA/core-maintainers : could you please help to review?

**Pull request merged**